test/rebar3_hex_integration_SUITE.erl

```diff
@@ -761,24 +873,7 @@ setup_mocks_for(docs, #{password := Password, password_confirmation := PasswordC
 
     expect_local_password_prompt(Password, PasswordConfirm),
 	expects_prompts([{any, string, "A", {returns, "A"}}, {"Proceed?", boolean, "Y", {returns, true}}]),
-
-    CocStr = "Before publishing, please read Hex CoC: https://hex.pm/policies/codeofconduct",
```

Comment:
Moved to helper function `default_publish_io/1`, not a great name, but will do for now.